# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../powerpc-utils/linux26.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual tweaking for 2.6 haeder files.

  - Rene <rene@exactcode.de>

--- ./backlight.c.orig	2004-10-14 14:28:03.000000000 +0200
+++ ./backlight.c	2004-10-14 14:28:11.000000000 +0200
@@ -24,7 +24,6 @@
 #include <sys/types.h>
 #include <linux/ioctl.h>
 #include <linux/types.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 #include <linux/pmu.h>
--- ./trackpad.c.orig	2004-10-14 14:27:39.000000000 +0200
+++ ./trackpad.c	2004-10-14 14:27:46.000000000 +0200
@@ -18,7 +18,6 @@
 #include <unistd.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 
--- ./vmode.c.orig	2004-10-14 14:35:18.000000000 +0200
+++ ./vmode.c	2004-10-14 14:40:58.000000000 +0200
@@ -4,7 +4,22 @@
 #include <sys/types.h>
 #include <sys/ioctl.h>
 #include <sys/vt.h>
-#include <asm/vc_ioctl.h>
+
+#define VC_GETMODE 0x7667
+#define VC_SETMODE 0x7668
+
+struct vc_mode {
+        int     height;
+        int     width;
+        int     depth;
+        int     pitch;
+        int     mode;
+        char    name[32];
+        unsigned long fb_address;
+        unsigned long cmap_adr_address;
+        unsigned long cmap_data_address;
+        unsigned long disp_reg_address;
+};
 
 int die(char *str)
 {
